media-libs/libtiger/libtiger-0.3.4-r1.ebuild

```diff
@@ -27,7 +27,9 @@ src_prepare() {
 
 multilib_src_configure() {
 	ECONF_SOURCE=${S} \
-	econf $(use_enable doc)
+	econf \
```

Comment:
Fixed. Can we get this finally merged, or is it fine to have x86 broken indefinitely?